### ISSUE TITLE
svc are not deleted when workspace is stopped or went idle since k8s v1.18.2

### ISSUE THREAD

**Issue by SDAdham** [eclipse-che/che#16871]
### Describe the bug

Since upgrading k8s to v1.18.2, regardless che's version 7.12.2, 7.12.1, 7.12.0 (it used to work on 7.12.0, 7.12.1 and older version of k8s - 1.16) and che doesn't cleanup after a workspace is stopped which causes the following error:
```
Error: Failed to run the workspace: "Failure executing: POST at: https://xx.xxx.xxx.xx/api/v1/namespaces/adham-che/services. Message: object is being deleted: services "db" already exists. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=null, kind=services, name=db, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=object is being deleted: services "db" already exists, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=AlreadyExists, status=Failure, additionalProperties={})."
```
I'll have to delete svc manually in order to be able to start the workspace backup again

### Che version

 - [ ] latest
 - [X] nightly
 - [ ] other: please specify
chectl/0.0.20200505-next.b843886 linux-x64 node-v10.20.1

### Steps to reproduce

Use k8s `1.18.2`, then startup a workspace, use it for a bit then stop then try to start it back up again

### Expected behavior

It should stop >> cleanup unused resources, i.e svc then we should be able to start the workspace backup again

### Runtime
   - [X] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)
v1.18.2

### Installation method
   - [X] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [X] other: please specify

**Comment by ibuziuk** [comment]
@skabashnyuk could you please take a look?

**Comment by skabashnyuk** [comment]
@SDAdham  can you please provide more che-server logs? What was the way how did you install che? what was `chectl` command?

**Comment by SDAdham** [comment]
@skabashnyuk : Yes, I used `che` how can I get che's logs?

**Comment by sparkoo** [comment]
@SDAdham it should be fixed in 7.13 (https://github.com/eclipse/che/pull/16729), can you please try it?

**Comment by SDAdham** [comment]
@sparkoo I can confirm that this issue is no longer happening since `7.13.1`. Thanks a lot!
